### DISCUSSION TITLE
Different Apps?

### DISCUSSION THREAD

**Discussion by Linux-Geezer** [dglent/meteo-qt#118] [Q&A] [answered]
What's the difference between this app and the Meteo app on GitLab: https://gitlab.com/bitseater/meteo?

**Comment by dglent** [comment, marked as answer]
What is common is that both get the data from openweathermap, meteo uses the GTK graphical framework, meteo-qt the Qt ...

**Reply by Linux-Geezer** [reply]
From the link..._Developed with Vala & Gtk_...face plant.  Apologies for not reading more closely. Thank you so much dglent. You've been very helpful and responsive.
